flixel/system/debug/FlxDebugger.hx

```diff
@@ -483,10 +483,15 @@ class FlxDebugger extends Sprite
 	@:allow(flixel.system.debug)
 	function onMouseFocusLost():Void
 	{
+		#if FLX_MOUSE
```

Comment:
Minor complaint, but it looks we just need a single `FLX_MOUSE` check for this entire function.

Reply:
Yep, no need for two checks.